ajax/libs/dygraph/package.json

```diff
@@ -1,28 +1,25 @@
 {
   "name": "dygraph",
-  "filename": "dygraph-combined.js",
-  "version": "1.1.1",
+  "filename": "dygraph.min.js",
+  "version": "2.0.0",
   "description": "dygraphs is a fast, flexible open source JavaScript charting library.",
   "homepage": "http://dygraphs.com/",
   "keywords": [
-    "graphs",
-    "charts",
-    "interactive"
+    "timeseries",
+    "chart",
+    "canvas",
+    "visualization",
+    "dygraphs"
+  ],
+  "npmname": "dygraphs",
+  "npmfilemap": [
```

Comment:
Since this is a high priority issue, so I just merged and fixed locally, please be careful next time.